addons/website_sale_stock_wishlist/static/src/js/website_sale.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
```

Comment:
The code in this file is extremely similar to the code in `addons/website_sale_stock/static/src/js/website_sale.js`
We should avoid that duplication and make it work in a more generic manner and just have this file add new events listener if possible.

Reply:
I made some changes to make it more generic. There should be no more duplicate code.